src/PhpSpec/Runner/Maintainer/CollaboratorsMaintainer.php

```diff
@@ -123,8 +123,8 @@ public function getPriority()
     private function generateCollaborators(CollaboratorManager $collaborators, \ReflectionFunctionAbstract $function, \ReflectionClass $classRefl)
     {
         if ($comment = $function->getDocComment()) {
-            $comment = str_replace("\r\n", "\n", $comment);
-            foreach (explode("\n", trim($comment)) as $line) {
+            //$comment = str_replace("\r\n", "\n", $comment);
+            foreach (explode(PHP_EOL, trim($comment)) as $line) {
```

Comment:
this means that any testsuite using LF line endings would now break on windows. It does not make any sense.